src/libs/actions/Search.ts

```diff
@@ -355,6 +364,13 @@ function unholdMoneyRequestOnSearch(hash: number, transactionIDList: string[]) {
     API.write(WRITE_COMMANDS.UNHOLD_MONEY_REQUEST_ON_SEARCH, {hash, transactionIDList}, {optimisticData, finallyData});
 }
 
+function unholdMoneyRequestOnMoneyRequestReport(transactionIDList: string[], reportActions: ReportAction[]) {
+    transactionIDList.forEach((transactionID) => {
+        const action = getIOUActionForTransactionID(reportActions, transactionID);
+        unholdRequest(transactionID, action?.childReportID ?? '');
+    });
```

Comment:
I had to change to this for loop because with `Search` specific hold/unhold wasn't reflected in list